src/dbally/iql_generator/prompt.py

```diff
@@ -85,3 +88,24 @@ def __init__(
     ],
     response_parser=_validate_iql_response,
 )
+
+IQL_GENERATION_TEMPLATE_AGGREGATION = PromptTemplate[IQLGenerationPromptFormat](
+    chat=(
+        {
+            "role": "system",
+            "content": "You have access to API that lets you query a database supporting SINGLE aggregation.\n"
```

Comment:
"an API", "a SINGLE aggregation"

I would probably replace "query a database supporting" with "query a database with" but of course when it comes to the prompt, the results of the benchamarks is what's important.